tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_CylindricalBlastWave.cpp

```diff
@@ -0,0 +1,230 @@
+  grmhd_variables(const tnsr::I<DataType, 3, Frame::Inertial>& x) const
+      noexcept {
+    return variables(x, grmhd_variables_tags<DataType>{});
+  }
+};
+
+void test_create_from_options() noexcept {
+  const auto cylindrical_blast_wave =
+      test_creation<grmhd::AnalyticData::CylindricalBlastWave>(
+          "  InnerRadius: 0.8\n"
+          "  OuterRadius: 1.0\n"
+          "  InnerDensity: 1.0e-2\n"
+          "  OuterDensity: 1.0e-4\n"
+          "  InnerPressure: 1.0\n"
+          "  OuterPressure: 5.0e-4\n"
+          "  MagneticField: [0.1, 0.0, 0.0]\n"
+          "  AdiabaticIndex: 1.3333333333333333333");
+  CHECK(cylindrical_blast_wave == grmhd::AnalyticData::CylindricalBlastWave(
+                                      0.8, 1.0, 1.0e-2, 1.0e-4, 1.0, 5.0e-4,
+                                      std::array<double, 3>{{0.1, 0.0, 0.0}},
+                                      1.3333333333333333333));
+}
+
+void test_move() noexcept {
+  grmhd::AnalyticData::CylindricalBlastWave cylindrical_blast_wave(
+      0.8, 1.0, 1.0e-2, 1.0e-4, 1.0, 5.0e-4,
+      std::array<double, 3>{{0.1, 0.0, 0.0}}, 1.3333333333333333333);
+  grmhd::AnalyticData::CylindricalBlastWave cylindrical_blast_wave_copy(
+      0.8, 1.0, 1.0e-2, 1.0e-4, 1.0, 5.0e-4,
+      std::array<double, 3>{{0.1, 0.0, 0.0}}, 1.3333333333333333333);
+  test_move_semantics(std::move(cylindrical_blast_wave),
+                      cylindrical_blast_wave_copy);  //  NOLINT
+}
+
+void test_serialize() noexcept {
+  grmhd::AnalyticData::CylindricalBlastWave cylindrical_blast_wave(
+      0.8, 1.0, 1.0e-2, 1.0e-4, 1.0, 5.0e-4,
+      std::array<double, 3>{{0.1, 0.0, 0.0}}, 1.3333333333333333333);
+  test_serialization(cylindrical_blast_wave);
+}
+
+template <typename DataType>
+void test_variables(const DataType& used_for_size) noexcept {
+  const double inner_radius = 0.8;
+  const double outer_radius = 1.0;
+  const double inner_density = 1.0e-2;
+  const double outer_density = 1.0e-4;
+  const double inner_pressure = 1.0;
+  const double outer_pressure = 5.0e-4;
+  const std::array<double, 3> magnetic_field{{0.1, 0.0, 0.0}};
+  const double adiabatic_index = 1.3333333333333333333;
+
+  const auto member_variables = std::make_tuple(
+      inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
+      outer_pressure, magnetic_field, adiabatic_index);
+
+  CylindricalBlastWaveProxy cylindrical_blast_wave(
+      inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
+      outer_pressure, magnetic_field, adiabatic_index);
+
+  // Note: I select random numbers in the range {{-1.1, 1.1}} so that
+  // sometimes the random points are in the transition region and sometimes
+  // in the fixed region.
+  pypp::check_with_random_values<
+      1, CylindricalBlastWaveProxy::hydro_variables_tags<DataType>>(
+      &CylindricalBlastWaveProxy::hydro_variables<DataType>,
+      cylindrical_blast_wave, "AnalyticData.GrMhd.TestFunctions",
```

Comment:
@geoffrey4444 it's failing to import the module you name here. Just name it `TestFunctions`

Reply:
Thanks @nilsdeppe!